relay/src/cli.rs

```diff
@@ -314,6 +319,57 @@ pub fn init_config<P: AsRef<Path>>(config_path: P, _matches: &ArgMatches) -> Res
     Ok(())
 }
 
+/// Manages the on-disk spool file.
+pub fn manage_spool(config: &Config, matches: &ArgMatches) -> Result<()> {
+    if let Some(matches) = matches.subcommand_matches("clear") {
+        let path = match matches.get_one::<String>("path") {
```

Comment:
I think this should work:

```suggestion
        let path = match matches.get_one::<PathBuf>("path") {
```